### PR TITLE
made it nightguard ready

### PR DESCRIPTION
**Pull request by Joerg-Schoemer**
Hi Timo,

I've added some useful log messages and changed the time interval to 5 minutes because https://github.com/nightscout/nightguard needs the interval for correct alarms.
They just count the values to estimate the times :-(

Kind regrads,
Jörg

### CODE REVIEW

**Review by timoschlueter — COMMENTED** [review]
Very nice additions and thanks for the refactoring! 
I added a few comments. Whats your thought on making the interval configurable? :)

File: index.js [hunk 1]
```diff
@@ -15,65 +16,79 @@ const logger = createLogger({
...
+    "api-secret": NIGHT_SCOUT_API_TOKEN,
     "User-Agent": USER_AGENT,
     "Content-Type": "application/json",
 }
 
-logger.info("Started")
-cron.schedule("* * * * *", () => {
-    main();
-});
+const schedule = "*/5 * * * *";
```
[the remark this answers is not in the document]
Comment (Joerg-Schoemer) [reply index.js:74]: 👍🏻

File: index.js [hunk 2]
```diff
@@ -15,65 +16,79 @@ const logger = createLogger({
...
-const API_URL = "api-eu.libreview.io"
-const USER_AGENT = "FreeStyle LibreLink Up Nightscout Uploader";
+const LIBRE_LINK_UP_URL = "api-eu.libreview.io"
 const LIBRE_LINK_UP_VERSION = "4.1.1";
 const LIBRE_LINK_UP_PRODUCT = "llu.ios";
 
+/**
+ * NightScout API
+ */
+const NIGHT_SCOUT_URL = process.env.NIGHTSCOUT_URL;
```
[the remark this answers is not in the document]
Comment (Joerg-Schoemer) [reply index.js:49]: The environment variable has to  be changed by the user when updating. I wouldn’t make changes affecting user changes on heroku.

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Joerg-Schoemer) [reply index.js:74]: Most of the people using Nightscout have a sensor from Dexcom which sends in 5’ interval. Freestyle Libre3 is the first with 1’ interval, but there is no need for such a short interval.
All  alarms are based on the 5’ by historical reason.

[the remark this answers is not in the document]
Comment (sebdei) [reply index.js:74]: If someone would ask me the existing time interval of 1 minute is good practice. I really want to have my data as soon as possible. As you said, libre3 is able to read data every minute so why should we limit writings to the capabilities of dexcom?

**Pull request merged**